test/public-stats.test.js

```diff
@@ -29,7 +30,7 @@ describe('public-stats', () => {
     await pgClient.query('DELETE FROM indexer_query_stats')
     await pgClient.query('DELETE FROM daily_deals')
     await pgClient.query('DELETE FROM retrieval_timings')
-
+    await pgClient.query('DELETE FROM daily_client_retrieval_stats')
```

Comment:
Please preserve the empty line; it serves as a visual delimiter, making it easier for me to orientate in long blocks of code.